doc/source/_templates/layout.html

```diff
@@ -1,11 +1,29 @@
 {% extends "!layout.html" %}
 
+{%- block extrahead %}
+<style>
+  div.top-scipy-org-logo-header {
+    background-color: #fafafa;
+    border-bottom: 1px solid #6c7a89; /* Aurometalsaurus */
+    margin-top: 8px;
+  }
+  div.top-scipy-org-logo-header img {
+    height: 100px;
+    padding-left: 27px;
+  }
+  div.spc-navbar .nav-pills > li > a {
+    background-color: #4d77cf; /* Han Blue */
+  }
+</style>
+{{ super() }}
+{% endblock %}
+
 {%- block header %}
 <div class="container">
-  <div class="top-scipy-org-logo-header" style="background-color: #a2bae8;">
+  <div class="top-scipy-org-logo-header">
     <a href="{{ pathto('index') }}">
-      <img border=0 alt="NumPy" src="{{ pathto('_static/numpy_logo.png', 1) }}"></a>
-    </div>
+      <img border=0 alt="NumPy" src="{{ pathto('_static/numpylogo.svg', 1) }}">
```

Comment:
"NumPy" should be text and colored by `#013243` similar to the home page

Reply:
Had no idea that github showed colors (`#FF0000`)!